src/OSPSuite.UI/Extensions/ToolTipControllerExtensions.cs

```diff
@@ -12,12 +12,18 @@ public static ToolTipController Initialize(this ToolTipController toolTipControl
          toolTipController.ShowBeak = false;
          toolTipController.AllowHtmlText = true;
          toolTipController.ImageList = imageListRetriever.AllImages16x16;
-         toolTipController.AutoPopDelay = 300000;
          toolTipController.InitialDelay = delayInMilliseconds;
+         toolTipController.Initialize();
          return toolTipController;
       }
 
-      public static ToolTipController For(this ToolTipController toolTipController, EditorContainer editorContainer)
+      public static ToolTipController Initialize(this ToolTipController toolTipController)
+      {
+          toolTipController.AutoPopDelay = 300000;
```

Comment:
What do you think, should we move all the other property setters into this one, except the `imageListRetriever` and `TOOL_TIP_INITIAL_DELAY`?

It probably makes sense. I'm not sure what they all do, but `Rounded` and whatever can all go in here too.

Reply:
discussed and we believe the properties are related to showing images in the tooltips and so, should remain where they are.

Reply:
213123123

Reply:
123123